styles/style.css

```diff
@@ -138,8 +140,13 @@ background-image: url("../images/header-bike.jpg");
   margin-bottom: 2rem; 
   color: white;
   margin: auto;
+  padding-left: 1rem;
+}
+.btn1{
+  background-color: #c05326;
+  margin-left: 1rem;
+  margin-top: 3rem;
 }
-
 
```

Comment:
Your other button is not visible on the screen, it should have  visible bg color which also would enhance UX experience
![image](https://github.com/user-attachments/assets/ba8ea90f-05de-4fb4-b501-a46148515a84)